### DISCUSSION TITLE
Linux AppImage

### DISCUSSION THREAD

**Discussion by trymeouteh** [ventoy/Ventoy#1239] [Ideas]
With the Linux GUI now released, an AppImage would be great since it will allow easy installation for all distros, its a portable app. balenaEtcher users an AppImage to flash ISOs also.

**Comment by its0x08** [comment]
@trymeouteh  are you asking for a USB maker for Ventoy or for an AppImage that will download different OS ISOs like the UNetbootin?
I'm not sure I understood the topic of the discussion.

**Reply by probonopd** [reply]
I think the most portable way is to actually put the Ventoy Live ISO onto the Ventoy disk, and boot into it to update Ventoy. That's how I am doing it, anyway.

**Comment by trymeouteh** [comment]
No, to release the Linux app as an appimage, not as a tarball with loose files inside.
